README.md

```diff
@@ -50,6 +50,12 @@ TODO:
 - [ ] `setACL`
 
 
+### Building with Bazel
+
+* Compiling: `bazel build //:zdb_zk`
```

Comment:
one little nit: `fdb_zk` :)

Reply:
doah!  Sorry, on it!  Also sorry on the delay, I have been traveling.